nextcord/types/channel.py

```diff
@@ -63,6 +64,7 @@ class ForumChannel(_ThreadedBaseChannel):
     default_reaction_emoji: NotRequired[Optional[DefaultReaction]]
     default_thread_rate_limit_per_user: NotRequired[int]
     available_tags: NotRequired[List[ForumTag]]
+    default_forum_layout: ForumLayoutType
```

Comment:
According to the relevant [Discord API Docs PR](https://github.com/discord/discord-api-docs/pull/5693), this attribute is optional.